render/go/renderer.go

```diff
@@ -197,6 +199,8 @@ func ToTitle(s string) (ret string) {
 
 func MustLoadGinServerTestConfig(configPath, openAPIPath string) config.Data {
 	config.ConfigFilePath = configPath
+	absCfgPath, _ := filepath.Abs(config.ConfigFilePath)
```

Comment:
Why are we ignoring errors here and in the line below?

Reply:
I was relying on the panic below. These errors would have meant Load can't work. But I've handled them explicitly. Should be more clear now.